privacyidea/api/radiusserver.py

```diff
@@ -127,9 +127,16 @@ def test():
     password = getParam(param, "password", required)
     dictionary = getParam(param, "dictionary",
                           default="/etc/privacyidea/dictionary")
-
-    r = test_radius(identifier, server, secret, user, password, port=port,
+    r = 0
+    for server in servers.split(','):
+        # TODO: IPv6 break the split
+        serveraddr = server.split(':')
+        if len(serveraddr) >= 2:
+            serverport = int(serveraddr[1])
+        else:
+            serverport = port
+        r = test_radius(identifier, serveraddr[0], secret, user, password, port=serverport,
                     dictionary=dictionary, retries=retries, timeout=timeout)
-    g.audit_object.log({'success': r > 0,
+        g.audit_object.log({'success': r > 0,
                         'info':  r})
     return send_result(r > 0)
```

Comment:
Hm, I am not sure, what the result should be here.

So if the first server fails, but the 2nd succeeds, we get a success.
If the first is successful and the 2nd fails, we get a fail.

Reply:
I return now the first response